cime/config/e3sm/machines/config_compilers.xml

```diff
@@ -1804,7 +1804,16 @@ ntel/x86_64/2013/composer_xe_2013/composer_xe_2013_sp1.3.174/mkl/include </base>
     <append>-L$ENV{NETCDF_C_PATH}/lib -lnetcdf -L$ENV{NETCDF_FORTRAN_PATH}/lib -lnetcdff -L$ENV{ESSL_PATH}/lib64 -lessl -L$ENV{NETLIB_LAPACK_PATH}/lib -llapack</append>
     <append MPILIB="!mpi-serial"> -L$ENV{PNETCDF_PATH}/lib -lpnetcdf -L$ENV{HDF5_PATH}/lib -lhdf5_hl -lhdf5 </append>
   </LDFLAGS>
+  <MPICC> mpicc </MPICC>
```

Comment:
These compiler name settings aren't necessary if they're the same as the "gnu" block above.

Reply:
I would like to be explicit and consistent for all compiler families on Summit. So, added them.